lib/beaker/hypervisor/fusion.rb

```diff
@@ -54,12 +55,65 @@ def provision
         end
         time = Time.now - start
         @logger.notify "Spent %.2f seconds resuming VM" % time
+
+        begin
+          try_ssh_connection(host.name, host['user'], host['ssh'])
```

Comment:
Is there a way to use the command to gather the desired information -- instead of issuing an I'm-calling-this-command-for-its-connection-testing-side-effects-only command?